runelite-client/src/main/java/net/runelite/client/plugins/inventorygrid/InventoryGridPlugin.java

```diff
@@ -43,12 +48,19 @@
 	@Inject
 	private InventoryGridOverlay overlay;
 
+	@Inject
+	private InventoryGridConfig config;
+
 	@Inject
 	private OverlayManager overlayManager;
 
+	@Getter(AccessLevel.PACKAGE)
+	private int dragDelay;
+
 	@Override
 	public void startUp()
 	{
+		onConfigChanged(null);
```

Comment:
Don't call subscriber methods directly. Create some method to be called from here and from `onConfigChanged()` which contains the code you want to run instead.